Sources/UnleashProxyClientSwift/Client/UnleashProxyClientSwift.swift

```diff
@@ -1,50 +1,27 @@
 import Foundation
 import SwiftEventBus
 
-// MARK: - Welcome
-struct FeatureResponse: Codable {
-    let toggles: [Toggle]
-}
-
-// MARK: - Toggle
-public struct Toggle: Codable {
-    public let name: String
-    public let enabled: Bool
-    public let variant: Variant?
-}
-
-// MARK: - Variant
-public struct Variant: Codable {
-    public let name: String
-    public let enabled: Bool
-    public let payload: Payload?
-}
-
-// MARK: - Payload
-public struct Payload: Codable {
-    public let type, value: String
-}
-
-public enum UnleashEvent: String, CaseIterable {
-    /// Emitted when UnleashClient is ready after finished first flag fetch
-    case ready
-    /// Emitted when toggles have been updated
-    case update
-    /// Emitted on metrics sent
-    case sent
-    /// Emitted when metrics failed to send
-    case error
-}
-
-
 @available(macOS 10.15, *)
 public class UnleashClientBase {
     public var context: Context
     var timer: Timer?
     var poller: Poller
     var metrics: Metrics
 
-    public init(unleashUrl: String, clientKey: String, refreshInterval: Int = 15, metricsInterval: Int = 30, disableMetrics: Bool = false, appName: String = "unleash-swift-client", environment: String? = "default", context: [String: String]? = nil, poller: Poller? = nil, metrics: Metrics? = nil, customHeaders: [String: String] = [:]) {
+    public init(
+        unleashUrl: String,
+        clientKey: String,
+        refreshInterval: Int = 15,
+        metricsInterval: Int = 30,
+        disableMetrics: Bool = false,
+        appName: String = "unleash-swift-client",
+        environment: String? = "default",
+        context: [String: String]? = nil,
+        poller: Poller? = nil,
+        metrics: Metrics? = nil,
+        customHeaders: [String: String] = [:],
+        bootstrap: Bootstrap = .toggles([])
```

Comment:
If a explicit Poller and bootstrap is passed in here, the bootstrap will be ignored in favour of one passed directly into the injected Poller. This is not optimal, but a significant change to the initialiser that would remove duplicate initialised values in presence of explicit Poller felt out of scope, and a more major and breaking change to the API